src/button/component.jsx

```diff
@@ -892,17 +892,59 @@ export const Button : Component<ButtonOptions> = create({
             type:     'function',
             required: false,
             get value() : Function {
+                let initialHeight;
+                const logInlineGuestOutOfViewPortOnlyOnce = once((data) => {
+                    info(`inline_guest_outside_viewport`, data);
```

Comment:
Can we make this event name generic? This is something which could be an issue for any integration with a variable vertical height.

Reply:
good point, I updated the code to address your point.